src/librehatti/settings.py

```diff
@@ -20,15 +20,15 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.mysql",
-        "NAME": "librehatti3",
-        "USER": "root",
-        "PASSWORD": "J@1diamond",
-        "HOST": "localhost",
-        "PORT": "",
+        "NAME": "librehatti",
```

Comment:
Please use env file to define such information. Create file for dev, qa and prod